src/chaintree/chaintree.spec.ts

```diff
@@ -66,4 +65,89 @@ describe('ChainTree', ()=> {
       expect((await resp).value).to.eql(true)
     })
 
+    it('grafts ownership through a DID', async ()=> {
+      const c = await Community.getDefault()
+
+      const parentKey = await EcdsaKey.generate()
+      const parentTree = await ChainTree.newEmptyTree(c.blockservice, parentKey)
+      // need to make sure the parentTree exists with the signers
+      await c.playTransactions(parentTree, [setDataTransaction("hi", "hi")])
+
+      const childKey = await EcdsaKey.generate()
+      const childTree = await ChainTree.newEmptyTree(c.blockservice, childKey)
+
+      await c.playTransactions(childTree, [setOwnershipTransaction([(await parentTree.id())!])])
+
+      childTree.key = parentKey
+
+      await c.playTransactions(childTree, [setDataTransaction("parentOwnsMe", true)])
+      const resp = childTree.resolveData("/parentOwnsMe")
+      expect((await resp).value).to.eql(true)
+    })
+
+    it('grafts DID-based ownership through an intermediary tree', async ()=> {
```

Comment:
```ChainTree
       grafts DID-based ownership through an intermediary tree:
     Error: error creating NewAddBlockRequest: error processing block (valid: false): 0 - error resolving owners: unknown auth type for &{Tip:bafyreiehey7ga45mfzmeemtinhtx7spnugbjwmmub3c3r3j7dh5ktvheea Store:0x1131e20}: *dag.Dag
```

Reply:
resolved with `v0.7.1` release which is pulled in here now 👍